agent/start.go

```diff
@@ -35,7 +36,15 @@ func StartDaemon(ctx context.Context) (*Client, error) {
 	}
 
 	cmd := exec.Command(os.Args[0], "agent", "run", logFile)
-	cmd.Env = append(os.Environ(), "FLY_NO_UPDATE_CHECK=1")
+
+	versionNum := buildinfo.Version().Pre[0].VersionNum
+
+	env := os.Environ()
+
+	env = append(env, fmt.Sprintf("DEV_VERSION_NUM=%d", versionNum))
```

Comment:
how about prefixing this env var name with `FLY_` ?

```suggestion
	env = append(env, fmt.Sprintf("FLY_DEV_VERSION_NUM=%d", versionNum))
```

Reply:
You're right, that's a good idea